packages/jupyterlab-starters/src/tokens.ts

```diff
@@ -14,6 +14,7 @@ export const API = URLExt.join(PageConfig.getBaseUrl(), 'starters');
 export const DEFAULT_ICON_NAME = `${NS}:default`;
 export const DEFAULT_ICON_CLASS = `jp-StartersDefaultIcon`;
 export const CATEGORY = 'Starters';
+export const RANK = Infinity;
```

Comment:
This strikes me as odd: it's not JSON-serializable... if omitted, i would just preserve the default behavior, e.g. 

```ts
launcher.add({
  // ...
  ...(starters[name].rank == null ?  {} : {rank: starters[name].rank})
})
```

Reply:
Done, see: https://github.com/deathbeds/jupyterlab-starters/pull/78